contrib/docker/Dockerfile

```diff
@@ -1,60 +1,50 @@
 FROM python:3.6
 
-MAINTAINER Xiao Hanyu <hanyu.xiao@shopeemobile.com>
-
-# Add a normal user
-RUN useradd --user-group --create-home --shell /bin/bash work
-
 # Configure environment
 ENV LANG=C.UTF-8 \
-    LC_ALL=C.UTF-8 \
-    HOME=/home/work
+    LC_ALL=C.UTF-8
 
-RUN apt-get update -y
+WORKDIR /home/work/incubator-superset
 
 # Install some dependencies
 # http://airbnb.io/superset/installation.html#os-dependencies
-RUN apt-get update -y && apt-get install -y build-essential libssl-dev \
-    libffi-dev python3-dev libsasl2-dev libldap2-dev
-
-RUN apt-get install -y vim less postgresql-client redis-tools
+RUN apt-get update -y \
+    && apt-get install -y build-essential libssl-dev \
+        libffi-dev python3-dev libsasl2-dev libldap2-dev \
+        vim less postgresql-client redis-tools
```

Comment:
I put a separate `RUN apt-get install -y vim less postgresql-client redis-tools` because all `vim less postgresql-client redis-tools` are convenient tools for development, these are not necessary for production, while other packages like `build-essential libssl-dev` are needed for  superset to run.

I think it's OK to merge here for development purpose.

Reply:
Indeed, I will separate them again to make things clearer and fix the broken comment links.